product/views/MiddlewareServerGroup.yaml

```diff
@@ -0,0 +1,79 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Middleware Server Groups
+
+# Menu name
+name: Middleware Server Groups
+
+# Main DB table report is based on
+db: MiddlewareServerGroup
+
+# Columns to fetch from the main table
+cols:
+- feed
+- name
+- profile
```

Comment:
didn't see this mentioned in other places in this report (like col order, for instance). is this ok?